control/state.py

```diff
@@ -269,6 +270,10 @@ def lock_omap(self):
         got_lock = False
         assert self.rpc_lock.locked(), "The RPC lock is not locked."
 
+        if not self.omap_state.ioctx:
```

Comment:
Shouldn't this throw an exception?

Reply:
@caroav we're about to exit anyway. I'll add an exception, just in case.